### PR TITLE
chore: remove outdated documentation&update readme

### PR DESCRIPTION
**Pull request by mokimo**
## Description

Removes outdated documentation and adapts the readme.md to feature the newest developer onboarding tutorial.
- `getting-started.md` features the `hlx demo` CLI command which has been removed and therefor is not usable anymore.
- `getting-started2.md` has some interesting stuff, but uses paradigms such as using a `header.md` and `footer.md` which have been superseded by using `decoratePages` + [that](https://main--acom-kit--auniverseaway.hlx.page/tutorial) new tutorial provides everything that's needed to start out with helix 

## Related Issue

No issue, but since this is a small documentation update - hopefully that's fine as PR directly

## Motivation and Context

I've ran into troubles onboarding with helix and the old documentation confused me more as it did good. I would assume other developers face the same issues.

## How Has This Been Tested?

Ran through the old tutorials realising they don't work anymore hence can be removed.

## Types of changes

- [x] documentation changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by tripodsan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ## TL;DR?
 
-* [Start Developing Your First Helix Project in 60 Seconds](https://www.project-helix.io/)
+* [Start Developing Your First Helix Project](https://main--acom-kit--auniverseaway.hlx.page/tutorial)
```
Comment (rofe) [inline README.md:9]: Note this tutorial needs to be updated to Helix 3.

**Review by rofe — APPROVED** [review]
Thanks for your help cleaning up! 

Also obsolete:
- Helix Project Board
- Video Walkthroughs
- ...

File: README.md (hunk 1 above)
Comment (mokimo) [reply README.md:9]: cc @auniverseaway

**Comment by mokimo** [comment]
> Also obsolete:
Helix Project Board
Video Walkthroughs
...

Will likely do a follow up at some point, preferably not just deleting but replacing stuff
